distribution/resources/bin/gateway.bat

```diff
@@ -205,6 +207,15 @@ REM Check JAVA availability
     )
     EXIT /B 0
 
+REM set the ballerina thread pool size
+:setBallerinaWorkerPoolSize
+    IF "%BALLERINA_MAX_POOL_SIZE%"=="" (
+        SET BALLERINA_MAX_POOL_SIZE=100
+    ) else (
+        ECHO Info: Ballerina thread pool size is set to %BALLERINA_MAX_POOL_SIZE%
```

Comment:
Do we need to echo this? Thread pool size is something very internal to even ballerina as I know.

Reply:
This is ok since we only print this if custom value is defined